hybrid/per_func_comp_main.c

```diff
@@ -0,0 +1,257 @@
+/***
+ * This example implements a simple cheri type allocator to show
+ * how cheri otype values are determined.
+ * Note: cheriintrin.h is not used in this example.
+ ***/
+
+#include "../include/print.h"
+#include "include/utils.h"
+#include <sys/sysctl.h>
+#include <sys/types.h>
+
+#define SANDBOX_CODE_SZ 1000 // approx. later modify it.
+
+// NOTE ------
+/// cheriintrin.h is commented out temporarily in this example //
```

Comment:
Why is it commented? In general, unless there's a good reason, I'm allergic to commented code.

Reply:
Since I was asked to make the first PR before completing.

Reply:
cheriintrin.h is mostly macros, and somehow I got func signatures conflicts, that i couldn't be bothered to resolve now. So, commented out cheriintrin.h and added a few lines from it, if necessary.